packages/ui/core/Button/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {fireEvent} from '@testing-library/react-native';
+import renderer from 'react-test-renderer';
+import {StyledTheme} from '@animavita/theme';
+
+import {Mount, ThemeContext} from '../../../tests/helpers';
+import Button from '../Button';
+
+describe('Button', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const cb = jest.fn();
```

Comment:
This isn't a good practice. You should either declare the variables needed inside each it or create a setup function to render the component.

e.g. 
```javascript
const setup = propOverrides => {
    const defaultProps = {
      onPress: jest.fn().mockName('onPress'),
      ...propOverrides,
    };

    const element = Mount(<Button {...defaultProps} />);
    );

    return {
      element,
    };
  };
```

Reply:
Really, I'll improve it. Thanks again!